src/LatestFollowActivity.jsx

```diff
@@ -56,28 +71,37 @@ return (
     <H2>Follow Activity</H2>
 
     <Items>
-      {follows.map((item, i) => (
-        <Item key={i}>
-          <Widget
-            src="${REPL_ACCOUNT}/widget/AccountProfile"
-            props={{ accountId: item.accountId, hideAccountId: true }}
-          />
-
-          <Text small bold>
-            <i className="bi bi-arrow-right"></i>
-            <br />
+      {profiles &&
+        follows.map((item, i) => (
+          <Item key={i}>
             <Widget
-              src="${REPL_MOB_2}/widget/TimeAgo${REPL_TIME_AGO_VERSION}"
-              props={{ blockHeight: item.blockHeight }}
+              src="${REPL_ACCOUNT}/widget/AccountProfile"
+              props={{
+                accountId: item.accountId,
+                hideAccountId: true,
+                profile: profiles[item.accountId]?.profile,
+              }}
             />
-          </Text>
 
-          <Widget
-            src="${REPL_ACCOUNT}/widget/AccountProfile"
-            props={{ accountId: item.value.accountId, hideAccountId: true }}
-          />
-        </Item>
-      ))}
+            <Text small bold>
+              <i className="bi bi-arrow-right"></i>
+              {/* <br />
+              <Widget
+                src="${REPL_MOB_2}/widget/TimeAgo${REPL_TIME_AGO_VERSION}"
+                props={{ blockHeight: item.blockHeight }}
+              /> */}
```

Comment:
Why are this lines commented?

Reply:
Good catch - I forgot to add this back. We don't currently have any way to avoid the HTTP requests here for rendering the follow timestamp, but it should only create 5 extra requests. This would be an example where Query API can eventually help out.